doc/user_guide/conf.py

```diff
@@ -335,8 +354,20 @@
 # epub_tocdup = True
 
 
+# -- Options for linkcheck -------------------------------------------------
+
+linkcheck_anchors = True
+
+# puma is using an old https server and that causes errors in linkcheck.
+# Despite its name 'collab.metoffice.../twiki' requires authenticated access.
+linkcheck_ignore = [r'https://puma.nerc.ac.uk/trac/GOcean',
+                    r'code.metoffice.gov.uk/trac/lfric/attachment/wiki/LFRicDocumentationPapers/lfric_documentation.pdf']
```

Comment:
This long line breaks pycodestyle

Reply:
Fixed. I also realised we still had a link that used "http://puma..." rather than "https" so I've fixed that.